status-payloads-spec.md

```diff
@@ -81,15 +84,19 @@ The protobuf description is:
 message ChatMessage {
```

Comment:
@corpetty 

Do you mean this payload section? Why wouldn't we?

L82-128 - do we need this with the next section? I updated it to latest status-go protocol commit, but a reference may be better suited if we define it in a table directly afterwards.